open-metadata-implementation/adapters/open-connectors/event-bus-connectors/open-metadata-topic-connectors/kafka-open-metadata-topic-connector/src/main/java/org/odpi/openmetadata/adapters/eventbus/topic/kafka/KafkaOpenMetadataEventProducer.java

```diff
@@ -99,6 +100,10 @@ private void publishEvent(String event) throws ConnectorCheckedException
         boolean                  eventSent = false;
         long                     eventRetryCount = 0;
 
+        if( producer == null){
```

Comment:
Formatting isn't following the convention already present in the code:

```java
if (producer == null)
{
   // ...
}
```

Reply:
If I can be super-picky (sorry!) I'd suggest the conditionals should be:

```java
if (producer == null)
{
   // ...
}
```

At the moment all the `if ...` statements look more like function calls with the `(` coming immediately after `if` without any space in-between, and instead a space between the `(` and the variable used for comparison -- just looks messy...